application/pom.xml

```diff
@@ -378,6 +384,12 @@
             <version>0.28.2</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.opentripplanner</groupId>
```

Comment:
I made a PR to change the package name to `org.opentripplanner.ojp` https://github.com/opentripplanner/ojp-java-model/pull/2

The `ruterbanken` company does not exist, it was an early version of Entur.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/6510/commits/015459fe2610403c91bae6ce3a3e95b88afd5e22